From a3a0af4fb1237bed0af75868073f9a63db8b1864 Mon Sep 17 00:00:00 2001
From: Alexis Ballier <aballier@gentoo.org>
Date: Thu, 7 Apr 2011 10:19:23 -0300
Subject: [PATCH] avutil: make aes and sha api public

The functions are already av_ prefixed and sha1 header is already provided.
Install libavutil/{aes,sha}.h

Signed-off-by: Michael Niedermayer <michaelni@gmx.at>
Signed-off-by: Luca Barbato <lu_zero@gentoo.org>
---
 libavutil/Makefile |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/libavutil/Makefile b/libavutil/Makefile
index 300acb1..baee496 100644
--- a/libavutil/Makefile
+++ b/libavutil/Makefile
@@ -3,6 +3,7 @@ include $(SUBDIR)../config.mak
 NAME = avutil
 
 HEADERS = adler32.h                                                     \
+          aes.h                                                         \
           attributes.h                                                  \
           audioconvert.h                                                \
           avassert.h                                                    \
@@ -33,6 +34,7 @@ HEADERS = adler32.h                                                     \
           random_seed.h                                                 \
           rational.h                                                    \
           samplefmt.h                                                   \
+          sha.h                                                         \
           sha1.h                                                        \
 
 BUILT_HEADERS = avconfig.h
-- 
1.7.2.5

